src/client/pages/Thema/Vergunningen/detail-page-content/WVOS.tsx

```diff
@@ -12,14 +13,24 @@ export function WVOSContent({
 }: {
   vergunning: VergunningFrontend<WerkzaamhedenEnVervoerOpStraat>;
 }) {
+  const decision = () => {
+    return {
+      ...commonTransformers.decision(vergunning),
+      content:
+        // TODO: Moeten we hier niet de decision (deels verleend / verleend?) van de vergunning gebruiken om te bepalen of we deze tekst tonen?
+        vergunning.werkzaamheden.length > 1 && vergunning.processed
+          ? 'In het Besluit ziet u voor welke werkzaamheden u een ontheffing heeft gekregen.'
+          : vergunning.decision,
+    };
```

Comment:
Eens. Al vraag ik me wel af waarom we dit alleen voor `> 1` doen en niet ook voor `=== 1` de enkelvoudvorm:

```suggestion
        const werkzaamheidOrWerkzaamheden = vergunning.werkzaamheden.length > 1 ? 'werkzaamheden ' : 'werkzaamheid';
        vergunning.decision === 'Zie besluit'
          ? 'In het Besluit ziet u voor welke ${werkzaamheidOrWerkzaamheden} u een ontheffing heeft gekregen.'
          : vergunning.decision,
    };
```

Reply:
Deze zin moet alleen getoond worden als het meerdere werkzaamheden betreft. Dan is deze check toch duidelijk? Als het 1 werkzaamheid betreft kunnen we gewoon Verleend of Niet verleend tonen.

Reply:
Ahh ik maakte een denkfout. Ik dacht dat dit ging om welke werkzaamheden waren verleend maar het gaat om voor welke werkzaamheden de aanvraag is gedaan. Dan is het logisch